hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/freon/TestContentGenerator.java

```diff
@@ -57,7 +61,9 @@ public void writeWithByteLevelWrite() throws IOException {
 
   @Test
   public void writeWithSmallBuffer() throws IOException {
-    ContentGenerator generator = new ContentGenerator(1024, 1024, 10);
+    ContentGenerator generator =
+        new ContentGenerator.Builder().seyKeySize(1024).setBufferSize(1024)
+            .setCopyBufferSize(10).build();
```

Comment:
typically builder classes are initialized one setter at a line. For example

```suggestion
        new ContentGenerator.Builder()
            .seyKeySize(1024)
            .setBufferSize(1024)
            .setCopyBufferSize(10)
            .build()
```

Reply:
Thanks for pointing it out!!
I have taken care of it across all the implementations !!